### PR TITLE
2017 Honda Ridgeline

### PR DESCRIPTION
**Pull request by energee**
This is very similar to the Pilot, dbc PR [here](https://github.com/commaai/opendbc/pull/77). Will need the person with Ridgeline to verify the TX message for the radar once they get their EON/giraffe set up. I am assuming it matches the Pilot for now.

### CODE REVIEW

**Comment by rbiasini** [comment]
Nice! @energee , let me know when it's ready to be reviewed.

**Comment by energee** [comment]
@rbiasini This is looking like its ready to be merged, can you send me the checklist again?

**Comment by rbiasini** [comment]
@energee , the checklist in the Toyota port pretty much applies to the Honda as well. 
https://medium.com/@comma_ai/openpilot-port-guide-for-toyota-models-e5467f4b5fe6

File: selfdrive/car/honda/hondacan.py [hunk 1]
```diff
@@ -92,21 +92,20 @@ def create_radar_commands(v_ego, car_fingerprint, idx):
...
+    elif car_fingerprint == CAR.ACURA_ILX:
+      msg_0x301 = "\x0f\x18\x51\x02\x5a\x00\x00"
+    elif car_fingerprint == CAR.PILOT:
+      msg_0x301 = "\x00\x00\x56\x02\x58\x00\x00"
+    elif car_fingerprint == CAR.RIDGELINE:
+      msg_0x301 = "\x00\x00\x56\x02\x57\x00\x00"
     commands.append(make_can_msg(0x300, msg_0x300, idx, 1))
-  elif car_fingerprint == CAR.PILOT:
-    msg_0x301 = "\x00\x00\x56\x02\x58\x00\x00"
-    commands.append(make_can_msg(0x300, msg_0x300, idx, 1))
```
[the remark this answers is not in the document]
Comment (rbiasini) [reply selfdrive/car/honda/hondacan.py]: Sounds good

**Comment by energee** [comment]
Squashed the port and put the dbc in another commit in case you want to throw that part away when merging

**Comment by BlaineHa** [comment]
The RTL-E and Black Edition have Honda sense with ACC and LKAS. Its the same as your existing ridgeline guy (I saw his utube video...very thrilled to see this come together)

**Comment by energee** [comment]
@BlaineHa you should join the slack for more details http://slack.comma.ai

**Comment by rbiasini** [comment]
Nice job guys, merging.

**Pull request merged**